model/post.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require("mongoose");
+
+// schema for posting blog
+const schema = new mongoose.Schema({
+    title: {
+        type: String,
+        required: true
+    },
+    content: {
+        type: String,
+        required: true
+    },
+    post: {
```

Comment:
What is `post` supposed to hold?

Reply:
this is now comments holding its ObejectId